x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/read_only_callout/index.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiCallOut, EuiButton } from '@elastic/eui';
+import React, { memo, useCallback, useState } from 'react';
+
+import * as i18n from './translations';
+
+const ReadOnlyCallOutComponent = () => {
+  const [showCAllout, setShowCallOut] = useState(true);
```

Comment:
Did you want the capitalization this way? `showCAllout` or just `showCallout`?

Reply:
😞